ppoprf/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rand_core_ristretto = { version="0.5.1", package="rand_core" }
 bitvec = "1.0.0"
 curve25519-dalek =  { version = "3.2.0", features = [ "serde" ] }
 criterion = "0.3.5"
-serde = "1.0.137"
+serde = "1.0"
```

Comment:
What error do you get? I see [brave-core locked at 1.0.136](https://github.com/brave/brave-core/blob/v1.43.26/build/rust/Cargo.lock#L2185), while adblock-rust [only requires 1.0.0](https://github.com/brave/adblock-rust/blob/v0.5.6/Cargo.toml#L28).

You should be able to `cargo update -p serde` in build/rust to move the brave-core build to a mutually compatible version.

Reply:
yep, that fixed my issue. will close this PR, thx!